frontend/lib/tests/shop/monobank-tx2-compensation.test.ts

```
@@ -0,0 +1,123 @@
+const selectMock = vi.fn(() => ({
+  from: () => ({
+    where: () => ({
+      limit: async () => [{ metadata: {} }],
+    }),
+  }),
+}));
+
+function makeUpdateQuery<T>(rows: T[]) {
+  return {
+    returning: async () => rows,
+    then: (resolve: any, reject: any) =>
```

Biome (2.3.14)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)